Decred Wallet/Features/Custom Dialogs/SimpleTextInputDialog.swift

```diff
@@ -59,13 +62,14 @@ class SimpleTextInputDialog: UIViewController {
         
         self.textField.addTarget(self, action: #selector(self.textFieldEditingChanged), for: .editingChanged)
         self.textField.delegate = self
+        self.textField.becomeFirstResponder()
         
         self.listenForKeyboardVisibilityChanges(delegate: self)
     }
 
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
-        self.textField.becomeFirstResponder()
+        self.textField.text = self.textFieldText
```

Comment:
Should only set this in view did load to avoid overwriting the user's text when this view appears (in cases where the user has changed the textfield text then the screen is locked/unlocked or something else causes this view did appear function to be triggered).

Reply:
this does not override users test its just for the animating the view